packages/migrator/src/index.ts

```diff
@@ -403,3 +403,57 @@ class RepairAction extends CommandLineAction {
 export interface RepairOptions {
   dryRun?: boolean
 }
+
+type LogMessage = Record<string, unknown>
+
+/**
+ * Logs messages to console. Known events are prettified to strings, unknown
+ * events or unexpected message properties in known events are logged as objects.
+ */
+export const prettyLogger: NonNullable<SlonikMigratorOptions['logger']> = {
+  info: message => prettifyAndLog('info', message),
+  warn: message => prettifyAndLog('warn', message),
+  error: message => prettifyAndLog('error', message),
+  debug: message => prettifyAndLog('debug', message),
+}
+
+function prettifyAndLog(level: keyof typeof prettyLogger, message: LogMessage) {
+  const MESSAGE_FORMATS: Record<string, (msg: LogMessage) => [string, LogMessage]> = {
```

Comment:
It looks like this object always has (effectively) the same value, but is created anew for each log. Can it be moved outside?

Reply:
Correct, it's now outside.